packages/grafana-ui/src/components/GraphNG/utils.test.ts

```diff
@@ -133,6 +146,9 @@ function mockDataFrame() {
   const f5Config: FieldConfig<GraphFieldConfig> = {
     displayName: 'Metric 4',
     decimals: 2,
+    color: {
```

Comment:
Had to change and add this as `getFieldColorModeForField` returns threshold as default if color.mode is not set.